public/angular/partials/slots.html

```diff
@@ -17,7 +17,7 @@ <h3>{{title}}</h3>
              drag="talk" dragStyle="columnDrag"
              dropStyle="columnDrop" title="{{talk.state.code}}">
             <span class="proposalTitle">{{talk.title}}</span>
-            <span class="badge badge-secondary">{{talk.id}}</span>
+            <a class="badge badge-secondary" href="/cfpadmin/proposal/{{talk.id}}" target="_blank">{{talk.id}}&nbsp;<i class="fas fa-external-link-alt"></i></a>
```

Comment:
It is not a good pattern to hard-code URLs in template. If possible, can you try to use the @routes ? If it is not possible, just let me know. Thanks

Reply:
I @gree @nicmarti  but here we are in angular (/public/angular/...) thus out of the scope of Play and I didn't find how to do it. Any idea ?

Reply:
ok for me then... And we cannot use Play JS router